react_front_end/src/__tests__/pages/GameInfoPage.js

```diff
@@ -1,5 +1,43 @@
+import "@testing-library/jest-dom";
+import React from "react";
+import { render, fireEvent, screen } from "@testing-library/react";
+import GameInfoPage from "../../pages/GameInfoPage";
+import {AppContext} from "../../App";
+import "@testing-library/jest-dom";
+
+
 describe("GameInfoPage", () => {
   it("should pass a smoke test", async () => {
```

Comment:
Rewrite this test to check that something is rendered. And rename the test. A smoke test is simply to make sure that the test runner is working.

Reply:
Hi @bernardbaker , this test will render the component, and will fail if there are any errors with the initialization. So, it's a smoke test of the GameInfoPage component, and not a smoke test of the test runner. I already found this test to be useful because it will fail if the GameInfoPage is rendered without wrapping it in the proper context, and it gives a better idea of where the component is failing, IE whether it didn't render correctly, or whether it failed a smoke test completely. I'll add a separate test to check the rendering